tests/tests.py

```diff
@@ -1321,7 +1417,8 @@ def test_ec2(self):
             LaunchConfigurationName=Ref("BaseHostLaunchConfig"),
             AvailabilityZones=GetAZs(""),
             HealthCheckGracePeriod=300,
-            HealthCheckType='EC2'
+            HealthCheckType='EC2',
+            DependsOn=['AttachGateway']
```

Comment:
This dependency is hardcoded rather than looked up. Although in our current state we only have the one gateway attachment (and thus the two are equivalent), is there is reason the other dependencies are looked up but this one not?